central/metrics/central.go

```diff
@@ -199,12 +201,29 @@ var (
 		Name:      "sensor_event_deduper",
 		Help:      "A counter that tracks objects that has passed the sensor event deduper in the connection stream",
 	}, []string{"status"})
+
+	pipelinePanicCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "pipeline_panics",
+		Help:      "A counter that tracks the number of panics that have occurred in the processing pipelines",
+	}, []string{"resource"})
 )
 
 func startTimeToMS(t time.Time) float64 {
 	return float64(time.Since(t).Nanoseconds()) / float64(time.Millisecond)
 }
 
+// IncPipelinePanics increments the counter tracking the panics in pipeline processing
+func IncPipelinePanics(msg *central.MsgFromSensor) {
```

Comment:
[nit] it's bothering me that this is not sorted with the rest of Increment methods...